packages/strapi-plugin-content-manager/admin/src/components/InputUID/RightLabel.js

```diff
@@ -0,0 +1,67 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { Success, Remove } from '@buffetjs/icons';
+import styled from 'styled-components';
+import { useGlobalContext } from 'strapi-helper-plugin';
+
+import pluginId from '../../pluginId';
+
+// This component is only used in this file so there is no need to create a separated file.
+const RightContentLabel = styled.div`
```

Comment:
I am not a huge fan of having two components into the same file but since this one is only used to apply your style it's ok plus I know that it will be replaced by the `Text` one later. However, let's try avoid it as much as possible.

> Note you don't need to create a specific file for this one as it will soon be replaced by the `Text` one so you can leave it in this file.